City-of-Helsinki/helsinki-paatokset
# PP-282: PP-282: fixed sorting and arranged parties alphabetically

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-282

Changed policymaker service to split names from first name field to first name and last name fields. Fixed sorting with parties and sorted parties in dropdown. Also fixed hover styles if link is missing.

**How to test**

Checkout branch and run `drush cr` and go to i.ex. https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto 
Test the functionality of the members block, filtering and sorting should work. If a link is missing there shouldn't be any hover styles (i.ex. Suldaan Said Ahmed shouldn't have hover effect.)
